src/module/rolls/chat-message.ts

```diff
@@ -7,21 +7,37 @@ import { SFMoveDataProperties } from '../item/itemtypes'
 import { ROLL_OUTCOME } from './roll'
 import { renderRollGraphic } from './roll-graphic'
 
-function rollResult(a: number, c1: number, c2: number): ROLL_OUTCOME {
+function resolveIronswornRoll(a: number, c1: number, c2: number): ROLL_OUTCOME {
   if (a <= Math.min(c1, c2)) return ROLL_OUTCOME.MISS
-  if (a > Math.max(c1, c2)) return ROLL_OUTCOME.STRONG
-  return ROLL_OUTCOME.WEAK
+  if (a > Math.max(c1, c2)) return ROLL_OUTCOME.STRONG_HIT
+  return ROLL_OUTCOME.WEAK_HIT
 }
 
-function outcomeText(outcome: ROLL_OUTCOME, match: boolean): string {
-  let key = 'WeakHit'
-  if (outcome === ROLL_OUTCOME.MISS) {
-    key = match ? 'Complication' : 'Miss'
+function momentumBurnIsValid(
+  rawOutcome: ROLL_OUTCOME | undefined,
+  momentumOutcome: ROLL_OUTCOME
+) {
```

Comment:
oh, yeah, that makes sense. another possibility is `momentumIsBurnable`, but that speaks more to the purpose of the check rather than what the function is actually _doing_. so i reckon `momentumBurnWouldUpgrade` is better.

aside: this is one reason why i kinda like a numeric enum for roll outcomes -- makes comparing them dead simple.